sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -394,6 +400,20 @@ export interface PhoneNumberEntity {
     status?: string;
 }
 
+// Warning: (ae-internal-missing-underscore) The name "PhoneNumberPollerClient" should be prefixed with an underscore because the declaration is marked as @internal
```

Comment:
Can we mark the `client` properties on the operations also `@internal`, and let's do the _ prefix